content/W2016/W16-event-big-data-panel.md

```diff
@@ -0,0 +1,25 @@
+Title: Big Dataa Panel
+Date: 2016-02-15
+Category: Events
+Tags: big data, panels
+Slug: big-data-panel-w16
+Author: Fatema Boxwala
+Summary: Learn about the world of Big Data, and the roles of various women 
+in that space!
+
+Our much anticipated Big Data Panel is finally here! 
+
+Come out to hear about the experiences of women working in the data world.
+Learn what Big Data really is, and what kind of work exists in that space 
+for you!
+
+We will be taking questions from the audience at the end of the panel so
+bring any that you have.
+
+
+## Event Details ##
+
++ **Who:** Everyone!
++ **What:** Big Data Panel
++ **Where:** DC 1304
```

Comment:
That room is tiny, it only fits 50 people (and it's cramped for that number). Is that large enough for what I'm assuming is intended to be a larger event?